lib/mongoid_rails_migrations/mongoid_ext/railties/database.rake

```diff
@@ -33,24 +39,30 @@ namespace :db do
 
   desc 'Current database version'
   task :version => :environment do
```

Comment:
Putting it here but it applies to others: most of these tasks load the whole rails `environment` but I don't think they all actually need it? if we can remove some of the we can make them a lot faster (not related to this MR of course, could just be a nice side improvement IF It does not require more changes)

Reply:
Good point, let's create a dedicated issue for this as this is not related to this MR.